variables.tf

```diff
@@ -75,6 +81,12 @@ variable "allow_cloudwatch" {
   type        = string
 }
 
+variable "allow_cloudfront" {
+  description = "Allow Cloudfront service to export logs to bucket."
+  default     = false
+  type        = string
```

Comment:
If it's `false`, why isn't it a `bool`?

Reply:
Because the others are strings? :o)

Reply:
yeah, we should at the very least switch this to bool and preferably the other allow_* variables as well

Reply:
Since I'm withdrawing these code changes, I can do that in a separate PR.